### ISSUE TITLE
No output in Jupyter notebook with R kernel in default image 

### ISSUE THREAD

**Issue by DrXyzzy** [sagemathinc/cocalc#3120]
1. In a project with `Default` compute image selected, open a Jupyter notebook with `R (R-Project)` kernel.
1. Attempt to execute the following code cell and observe that no output is seen.
    ```
    mean(c(1,2,3,4,99))
    ```
1. Switch to `Old image` and restart the project.
1. Open the same Jupyter notebook. Click Halt. Reopen. Run the cell and observe that there is output.

Note that with the `Default` image, the same R code produces output with Jupyter console in a .term:
```
~$ jupyter console --kernel=ir
...
In [1]: mean(c(1,2,3,4,99))
[1] 21.8
```

**Comment by haraldschilly** [comment]
I can't reproduce it, although I saw similar situations with a stuck notebook right after the project was restarted.

**Comment by williamstein** [comment]
There were page(s) of scary code in the old Jupyter backend code related to making sure the kernel doesn't get stuck.  I could see no point in it (it didn't help at all in my test suite) and I did not port it over during the typescript rewrite.   So... maybe I somehow caused this new bug recently. 

For what it is worth, when I live demoed CoCalc yesterday at JupyterCon in a brand new project, everything went perfectly..

**Comment by DrXyzzy** [comment]
I have two projects running the default image. 
- [this one](https://cocalc.com/projects/ccd4d4a4-29a8-4c39-85c2-a630cb1e9b6c/files/ISSUES/simple-r.ipynb?session=default) runs the simple example above and gets output every time
- [this one](https://cocalc.com/projects/db982efa-e439-4e2d-933b-7c7011c6b21a/files/Public/simple-r.ipynb?session=default) shows no output every time. Executing the cell causes the circle with dot to appear between brackets, then circle without dot, then back to empty space between brackets, i.e. no execution number. And focus has moved to the following cell.

Weird part is, the project that now works failed 100% of the time last night. Both projects have been restarted multiple times, as well as multiple iterations of browser refresh and restart, and tabs closed and reopened.

UPDATE: I restarted the project where output was being produced. Then I halted and opened the .ipynb with the simple test. The pane for the body of the notebook went white except for "Loading..." at upper left, then the project restarted **again**. Now the simple notebook is not producing output in that project. Still with Default image. I was not exceeding memory or disk space bounds or running other processes in the project at this time.
